client/post-editor/media-modal/gallery/index.jsx

```diff
@@ -114,6 +114,13 @@ const EditorMediaModalGallery = React.createClass( {
 		this.props.onUpdateSettings( updatedSettings );
 	},
 
+	reverseOrder( ) {
+		const reverseItems = reverse( this.props.settings.items.slice( 0 ) );
```

Comment:
Nice catch on the [mutation note on `_.reverse`](https://lodash.com/docs/4.16.4#reverse) and the need to create a clone of the array 👍 

You could alternatively use newer ES6 semantics to make it more condensed:

```js
const reverseItems = reverse( [ ...this.props.settings.items ] );
```

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_operator